### PR TITLE
fix: use "⌘" as modifier key name on macOS

### PR DESCRIPTION
**Pull request by shuuji3**
This replaces the wrong "Ctrl" with "Command" as a modifier key name in the command pallet on macOS.

**Before**
Screenshot 2023-01-29 at 2 55 46

**After**
Screenshot 2023-01-29 at 2 55 55

I used `navigator.userAgentData.platform` to detect the platform name. However, this is implemented only in Chromium-based browsers at this moment.

ref. ["userAgentData" | Can I use... Support tables for HTML5, CSS3, etc](https://caniuse.com/?search=userAgentData)

If we want to work this on any browser, we need to use another way until all major browsers implement `userAgentData`. Then, does anyone know the best way to reliably detect the macOS in the Vue ecosystem?

### CODE REVIEW

**Comment by shuuji3** [comment]
It seems like TypeScript doesn't allow to use newer attribute `userAgentData`.

**Comment by userquin** [comment]
@shuuji3 you can use `useIsMac`, you can find it on `composables/misc.ts` module

**Comment by shuuji3** [comment]
@userquin Thanks, that's exactly what I looked for and somehow missed finding.

File: components/command/CommandPanel.vue [hunk 1]
```diff
@@ -64,6 +65,8 @@ const result = $computed<QueryResult>(() => commandMode
   : searchResult,
 )
 
+const modifierKeyName = $computed(() => useIsMac() ? 'Command' : 'Ctrl')
```
Comment (userquin) [inline components/command/CommandPanel.vue:68]:
```ts
const isMac = useIsMac()
const modifierKeyName = $computed(() => isMac .value ? 'Command' : 'Ctrl')
```

File: components/command/CommandPanel.vue [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <script setup lang="ts">
+import { useIsMac } from '~/composables/misc'
```
Comment (userquin) [inline components/command/CommandPanel.vue:2]: you don't need to import the module, we've auto import enabled

**Comment by shuuji3** [comment]
Thanks for reviewing. Applied your suggestion.

File: components/command/CommandPanel.vue [hunk 3]
```diff
@@ -64,6 +64,9 @@ const result = $computed<QueryResult>(() => commandMode
   : searchResult,
 )
 
+const isMac = useIsMac()
+const modifierKeyName = $computed(() => isMac.value ? 'Command' : 'Ctrl')
```
Comment (danielroe) [inline components/command/CommandPanel.vue:68]:
```suggestion
const modifierKeyName = $computed(() => isMac.value ? '⌘' : 'Ctrl')
```

Comment (shuuji3) [reply components/command/CommandPanel.vue]: That's a good idea.🙂  Here's the new screenshot:

Screenshot 2023-01-29 at 15 50 01

**Comment by shuuji3** [comment]
Thanks for merging!

**Pull request merged**